etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-access.R

```diff
@@ -189,18 +189,17 @@ if (!aws.s3::object_exists(remote_file_walk_warehouse)) {
   tmp_file_walk <- tempfile(fileext = ".geojson")
   aws.s3::save_object(remote_file_walk_raw, file = tmp_file_walk)
 
-  temp <- st_read(tmp_file_walk) %>%
+  st_read(tmp_file_walk) %>%
     st_transform(4326) %>%
+    rename(
+      walkability_rating = Walkabilit,
+      amenities_score = Amenities,
+      transitaccess = TransitAcc
+    ) %>%
```

Comment:
This `rename` chunk started throwing an error for seemingly no reason, which resolves by shifting the code up a few lines.